nix/README.md

```diff
@@ -0,0 +1,50 @@
+# Nix build support facilities
+
+## Development Environment
+
+The dependencies for a dynamic built can be fetched with
```

Comment:
As a reader, it would be nice to get a little intro paragraph on what this is about and why I want it. Especially because nix is so mysterious for most people.

Reply:
I don't think we should do this in our own docs. Do we explain why the user should use apt or brew? no.